docs/developer_guides/pipelines/dataparsers.md

```diff
@@ -145,7 +145,7 @@ See the code!
 
 ###### Record3D
 
-This dataparser can use recorded data from a >= iPhone 12 Pro using the Record3D app. Record a video and export with the `EXR + JPG sequence` format. Unzip export and `rgb` folder before training.
+This dataparser can use recorded data from a >= iPhone 12 Pro using the [Record3D app](https://record3d.app/) . Record a video and export with the `EXR + JPG sequence` format. Unzip export and `rgb` folder before training.
```

Comment:
maybe also add link directly to custom dataset part so that ppl know where to route to if they are unfamiliar with record3d

Reply:
e.g. add a seealso admonition with link to custom_dataset.